basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,59 @@
 const helpers = require('./project-1');
 
+describe('multiplication helpers', () => {
+    describe('multiplyByTen', () => {
+        it('should multiply a number by ten', () => {
+            const expected = 350;
+            
+            const actual = multiply(35, 10);
+        
+
+            expect(actual).toEqual(expected);
+        
+        });
+    });
+});    
+
+describe('subtractFive', () => {
+    it('should subtract a number by five', () => { 
+        const expected = 95;
+        const actual = subtract(100, 5);
+
+        expect(actual).toEqual(expected);
+
+    })
+
+});
+
+
+describe('areSameLength', () => {
+    it('Should return strings that are the same length', () => {
+
+        const expected = 'si','no';
+        const actual = areSameLength(str1, str2);
+
+        expect(actual).toEqual(expected);
+    })
+
+});
+
+
+
+//FUNCS
+function multiply(a, b) {
```

Comment:
Are you confused on the expectations? I'd like you to reference `helpers` to access your functions.

More than anything we want to identify any unpredictable outcomes before someone else does so we can make the adjustments necessary to handle them gracefully. We already know they work with valid data, how do they work when we try things that might not work?